addons/docs/docs/mdx.md

```diff
@@ -0,0 +1,180 @@
+<center>
+  <img src="./media/hero.png" width="100%" />
+</center>
+
+# Storybook Docs MDX
+
+> ⚠️ MDX support is an experimental feature in Storybook 5.2. The API may change in 5.3 outside of the normal semver rules. Be forewarned!
+
+`MDX` is the syntax [Storybook Docs](../README.md) uses to capture long-form markdown documentation and stories in one file. You can also write pure documentation pages in `MDX` and add them to Storybook alongside your stories.
+
+- [Basic example](#basic-example)
+- [MDX-Flavored CSF](#mdx-flavored-csf)
+- [Writing stories](#writing-stories)
+- [Embedding stories](#embedding-stories)
+- [Decorators and parameters](#decorators-and-parameters)
+- [More resources](#more-resources)
+
```

Comment:
Worth it to re-state the regex change for the config file to prevent confusion if folks land here from Google or elsewhere.